common/events.h

```diff
@@ -165,6 +165,10 @@
 #define SPECENUM_VALUE133 E_UNIT_ACTION_TARGET_OTHER
 #define SPECENUM_VALUE134 E_UNIT_ACTION_TARGET_HOSTILE
 #define SPECENUM_VALUE135 E_UNIT_WAKE
+// --
```

Comment:
```suggestion
// Combat without winner (Combat_Rounds)
```

Reply:
Should these specenum values all pushed down by 1 (or 2) so the new TIE ones can live next to their WIN and LOST counterparts?

Reply:
This would screw up the icons for all changed events in old clients, so... no :sweat_smile: